allennlp/modules/seq2seq_encoders/__init__.py

```diff
@@ -6,17 +6,17 @@
 
 The available Seq2Seq encoders are
 
-- `"gru"` : https://pytorch.org/docs/master/nn.html#torch.nn.GRU
-- `"lstm"` : https://pytorch.org/docs/master/nn.html#torch.nn.LSTM
-- `"rnn"` : https://pytorch.org/docs/master/nn.html#torch.nn.RNN
+- `"gru"` : https://pytorch.org/docs/master/generated/torch.nn.GRU.html#torch.nn.GRU
```

Comment:
fde5404bddb
Fix link to PyTorch modules.
BTW, I'm wondering if these should refer to `allennlp.modules.seq2seq_encoders.pytorch_seq2seq_wrapper.*`.

Reply:
Yea, I think agree with you there. Feel free to make that change if you want.

Reply:
Updated in c0e440a.